decidim-core/app/packs/stylesheets/decidim/vizzs/_areachart.scss

```diff
@@ -12,17 +12,20 @@
   }
 
   .area {
-    fill: rgba(var(--primary-rgb), 0.05);
+    fill: var(--primary);
+    fill-opacity: 0.05;
```

Comment:
Do we need these changes in the SCSS files after fixing the RGB values?

Or is there some benefit separating these values?

Reply:
OK, maybe then let's revert these changes?

While at it, I would then also revert the change done at #10424.

There would be actual benefit from reverting that because it would increase the contrast of the `X` mark within that button since now it also gets opacity.

Would it be possible to incorporate that change into this PR as well?

Reply:
Reverted in [207494d](https://github.com/decidim/decidim/pull/10688/commits/207494dd6a2c9ba884e9e5ba60ed797e6bc050ee)